### PR TITLE
Get sort from user preferences for table-export

### PR DESCRIPTION
**Pull request by alexabird**
Untitled copy
TESTING:
Upload multiple products, and run a * search. Click export > select CSV. Confirm that the export file contains the expected results and they they are in the same order as in the Results pane. Try a few different sorting configurations.

### CODE REVIEW

**Review by jrnorth — APPROVED** [review]

**Comment by alexabird** [comment]
build now

**Review by pvargas — APPROVED** [review]

**Review by swirepe — APPROVED** [review]

**Comment by sgalla** [comment]
With your changes, will the exported results respect the query sort order if there are not any result sorts set?

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx [hunk 1]
```diff
@@ -81,6 +82,9 @@ function getColumnOrder(): string[] {
 function getHiddenFields(): string[] {
   return user.get('user').get('preferences').get('columnHide')
 }
+function getSorts() {
+  return user.get('user').get('preferences').get('resultSort')
```
Comment (sgalla) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx]: With this change, will the exported results be sorted according to the query sorts if the user has not set any result sorts?

Comment (alexabird) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/table-export/table-export.tsx]: It does now 🙃 thanks for catching that!

**Pull request merged**